src/services/user/hooks/userService.js

```diff
@@ -15,7 +15,11 @@ const {
 } = require('../../../utils');
 
 const constants = require('../../../utils/constants');
-const { CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS, SC_DOMAIN } = require('../../../../config/globals');
+const {
+	CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS, SC_DOMAIN, HOST, SC_SHORT_TITLE,
+} = require('../../../../config/globals');
+
+const globalHooks = require('../../../hooks');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['globalHooks' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5424589)